scripts/ci-report.sh

```diff
@@ -11,7 +11,7 @@ mkdir -p "$ROOT_DIR/ci-report"
 cd "$ROOT_DIR/ci-report"
 
 # Disable until we find how some CI envs are being included
-# journalctl --since="$CI_REPORT_START_DATE" -o short-precise > journalctl.txt
+journalctl --since="$CI_REPORT_START_DATE" -o short-precise > journalctl.txt
```

Comment:
What if we just sed the token out here as discussed on the advisory?

Reply:
We could sed it too as an extra measure. But I believe with the above approach we have already addressed the root cause for the problem and ensure the gh_token is not logged in journalctl.
I will modify the above with 
journalctl --since="$CI_REPORT_START_DATE" -o short-precise | \
sed -E 's/ghs_[^[:space:]]+/ghs_***/g' > journalctl.txt

Reply:
With the above you ensure it won't be logged with the nix-installer action, but as we modify the workflow, other actions may eventually do the same. Use sed would be a way of removing it completely.
With the sed I think we can also skip the changes to the installer, and just let it be as is.

Reply:
Oh, I just pushed the commit with the change. So you want me to remove the previous changes as well or keep it as is since I am reducing the permissions on token as well with the previous change ?

Reply:
I think reducing the permissions we should, and as a rule of thumb we should always try to set as minimal as possible from now on.
The setting of token to "" and then resetting I think we can probably remove now, but we can let others review and comment as well. CC @Abhinandan-Purkait @niladrih

Reply:
The current changes look good to me.

Reply:
The changes seem fine to me.

Reply:
As discussed, we'll keep th entire change (root cause fix, permissions and post fix-journalctl). I am going ahead and merging the PR.